docs/content/integrations/snowflake/using-snowflake-with-dagster.mdx

```diff
@@ -198,16 +201,23 @@ def iris_dataset() -> pd.DataFrame:
         ],
     )
 
+    with snowflake.get_connection() as conn:
+        conn.cursor.execute("CREATE TABLE iris.iris_dataset AS SELECT * FROM iris_df")
```

Comment:
reminder to update this code snippet to based on https://github.com/dagster-io/dagster/pull/19744/files#r1536218001